### DISCUSSION TITLE
Accessibility (BT-754, BT-755)

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#403] [Q&A] [answered]
Hello Labs,

I have a question/remark about Accessibility (BT-754, BT-755).

I found this link about the topic:
https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html#accessibilitySection
"BT-754 is not repeatable, as the values in the codelist are mutually exclusive."

We use SDK 1.6.0.
BT-754, BT-755 -> Their parent node is "ND-AccessibilityJustification".
This node is repeatable -> "repeatable" : true.

Is SDK correct?

Thanks
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi Anna,
Your questioning is welcome. Looking a the codes from the Accessibility codelists, they seem to be mutually exclusive. Looking at the Regulation Annex BT-754 is identified as repeatable and having more than one occurrence of it requires the existence of more than one ND-AccessibilityJustification parent node as this parent node may only contain one code.
I will raise an issue for the Business people to clarify this.
KR,

**Reply by YvesJo** [reply]
Hi again,
After clarification, decision was made to have it non-repeatable in a next release.
KR.

**Reply by AnnaKSZ** [reply]
Thanks a lot!
